boards/x86/galileo/Kconfig.defconfig

```diff
@@ -40,7 +40,7 @@ config GPIO_PCAL9535A_0_DEV_NAME
 config GPIO_PCAL9535A_0_I2C_ADDR
 	default 0x25
 config GPIO_PCAL9535A_0_I2C_MASTER_DEV_NAME
-	default "$(dt_str_val,DT_SNPS_DESIGNWARE_I2C_90007000_LABEL)"
+	default "$(dt_str_val,DT_SNPS_DESIGNWARE_I2C_0_LABEL)"
```

Comment:
You can reference DTS values in kconfig?  I had no idea.

Reply:
Was added by @galak. See `scripts/kconfig/kconfigfunctions.py`.